README.md

```diff
@@ -48,9 +48,12 @@ in these getting started guide.
 > you will need to deploy the infrastructure using an approach that ensures
 > the services are scalable and reliable. For instance, using cloud provider
 > services such as Google Kubernetes Engine (GKE) for provisioning the Kubernetes cluster.
+> See "Installing prerequisites on Kubernetes using the init chart" section for a recommended production installation guide.
 >
 > The following guides are tested on Linux and MacOS.
 
+For a Kubernetes based setup, see 
```

Comment:
Incomplete statement?

Reply:
ah, good catch, i edited the top line but forgot to remove this.